manifests/firewalld.pp

```diff
@@ -0,0 +1,14 @@
+# allow connection on port 80 using https://github.com/voxpupuli/puppet-firewalld
+#
+# @api private
+class pulpcore::firewalld {
+
+  include ::firewalld
+
+  firewalld_service { 'allow http service':
+    ensure => 'present',
+    service => 'http',
+    zone => 'public',
+  }
+
+}
```

Comment:
I'm not sure this is needed. We've not had to use firewalld rules in any other modules.

Reply:
I included it because the ansible installer does:

   https://github.com/pulp/ansible-pulp/blob/master/roles/pulp-webserver/tasks/firewalld.yml

But we can remove it if it's not needed here